resources/function_help/json/array_concat

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "array_concat",
```

Comment:
I've been asked to match the postgres function names. Mistake on my side. I'll change it.